pkg/types/types.go

```diff
@@ -57,3 +57,8 @@ type ResourceServerPort interface {
 type SignalNotifier interface {
 	Notify() chan os.Signal
 }
+
+// RdmaFinder used to find the rdma devices
+type RdmaFinder interface {
```

Comment:
i think this interface represents a device's RDMA spec right ?

you could name the interface : `RdmaDeviceSpec`
it would then have a `Get(pciAddress string)` method to retrieve ther 

WDYT ?

Reply:
Seems good, adopted